lib/simple_form/inputs/money_input.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module SimpleForm
+  module Inputs
+    class MoneyInput < Base
```

Comment:
Another approach would be handling it inside of `StringInput`. But I created a new input to make it explicit. Does it make sense? 🤔